h2o-docs/src/product/data-munging/target-encoding.rst

```diff
@@ -146,136 +216,151 @@ Now we will perform target encoding on ``addr_state`` to see if this representat
-    ## 6         AK    0               0.1212239
+-  ``holdout_type="kfold"``
+-  ``blended_avg=TRUE``
+-  ``noise=0.2``
 
-For our testing data, we will use the parameters:
+.. example-code::
+   .. code-block:: r
 
--  ``holdout_type``: "None"
--  ``blended_avg``: False
--  ``noise_level``: 0
+    ext_train <- h2o.target_encode_transform(train, x = list("addr_state"), y = response, 
+                                             target_encode_map = te_map, holdout_type = "kfold",
+                                             fold_column="fold", blended_avg = TRUE, 
+                                             inflection_point=3, smoothing=1, seed = 1234)
 
-We do not need to apply any of the overfitting prevention techniques since our target encoding map was created on the training data, not the testing data.
+   .. code-block:: python
 
-.. code:: r
+    encoded_train = te_map.transform(frame=train, holdout_type="kfold", seed=1234)
 
-    ext_test <- h2o.target_encode_apply(test, x = list("addr_state"), y = response,
-                                        target_encode_map = te_map, holdout_type = "None",
-                                        fold_column = "fold",
-                                        blended_avg = FALSE, noise_level = 0)
+**Apply Target Encoding to Testing Dataset**
 
-    head(ext_test[c("addr_state", "TargetEncode_addr_state")])
+We do not need to apply any of the overfitting prevention techniques because our target encoding map was created on the training data, not the testing data.
 
-::
+-  ``holdout_type="none"``
+-  ``blended_avg=FALSE``
+-  ``noise=0`` 
+
+.. example-code::
+   .. code-block:: r
+
+    ext_test <- h2o.target_encode_transform(test, x = list("addr_state"), y = response,
+                                            target_encode_map = te_map, holdout_type = "none",
+                                            fold_column = "fold", noise = 0,
+                                            blended_avg = FALSE, seed=1234)
+
+   .. code-block:: python
+
+    # Add a fold column to the test set
+    fold = test.kfold_column(n_folds=5, seed=1234)
+    fold.set_names(["fold"])
+    test = test.cbind(fold)
+
+    # Create, fit, and apply (transform) the target encoding
+    ext_test = TargetEncoder(x=predictor, y=response, fold_column="fold", blended_avg=False, noise=0, seed=1234)
+    ext_test.fit(test)
+    ext_test.transform(frame=test, holdout_type="none", seed=1234)
 
-    ##   addr_state TargetEncode_addr_state
-    ## 1         AK               0.1521739
-    ## 2         AK               0.1521739
-    ## 3         AK               0.1521739
-    ## 4         AK               0.1521739
-    ## 5         AK               0.1521739
-    ## 6         AK               0.1521739
 
 Train Model with KFold Target Encoding
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Train a new model, this time replacing the ``addr_state`` with the ``TargetEncode_addr_state``.
```

Comment:
@angela0xdata this is a prefix from old R version of target encoding. I'm using `addr_state_te` naming.